packages/integration-tests/projects/suite-web/tests/suite/bug-report-form.test.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+
+// @group:suite
+// @retry=2
+
+describe('Stories of bug report forms', function () {
+    beforeEach(function() {
+        cy.task('startEmu', { wipe: true, version: '2.4.3' });
+        cy.task('setupEmu', { mnemonic: 'all all all all all all all all all all all all' });
+        cy.task('startBridge');
+
+        cy.viewport(1080, 1440).resetDb();
+        cy.prefixedVisit('/');
+
+    });
+
+    // todo: this test should fill report bug in Suite Guide
+    it(`Open Report form, fill bug report, submit report`, function () {
```

Comment:
For consistency with the rest of the code-base it would be better to use arrow function like:


```javascript
    it(`Open Report form, fill bug report, submit report`, () => {
```